layers/generate_layers.py

```diff
@@ -125,9 +142,76 @@ def generatePolygonAmenity(osm_input, amenity, filename):
     )
 
 
+def generateLayerBasedOnTwoPartTag(osm_input, tag_part_one, tag_part_two, filename):
+    if not osm_input:
+        raise Exception("You must specify --osm_input")
+
+    tmp = f"tmp_{filename}"
+    ensureEmptyTempDirectoryExists(tmp)
+
+    # First extract a .osm.pbf with all {tag_part_one}={tag_part_two} features
+    run(
+        [
+            "osmium",
+            "tags-filter",
+            osm_input,
+            f"nwr/{tag_part_one}={tag_part_two}",
+            "-o",
+            f"{tmp}/extract.osm.pbf",
+        ]
+    )
+
+    generateGeojsonAndPmTiilesFromOSMFile(f"{tmp}/extract.osm.pbf", filename)
+
+
+def generateGreenSpacesLayer(osm_input):
```

Comment:
See the PR in atip... I think the result of this doesn't add value beyond what the basemap shows, and refining it will take mroe work